src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/MainResources_de.properties

```diff
@@ -86,7 +86,6 @@ MSG_BundlerFailed=Fehler: Bundler "{1}" ({0}) konnte kein Package generieren
 MSG_BundlerConfigException=Bundler {0} aufgrund eines Konfigurationsproblems übersprungen: {1} \nEmpfehlung zur Behebung: {2}
 MSG_BundlerConfigExceptionNoAdvice=Bundler {0} aufgrund eines Konfigurationsproblems übersprungen: {1}
 MSG_BundlerRuntimeException=Bundler {0} nicht erfolgreich. Grund: {1}
-MSG_BundlerFailed=Fehler: Bundler "{1}" ({0}) konnte kein Package generieren
```

Comment:
Even though there was no change in the original, it looks like they removed this entry because it was a duplicate. We should consider filing a separate issue to have the duplicate removed in the original as well.